src/software/thunderscope/robot_diagnostics/chicker.py

```diff
@@ -234,15 +214,13 @@ def refresh(self):
         self.power_label.setText(str(power_value))
 
         # refreshes button state based on enable boolean
-        self.change_button_state(self.kick_button, self.buttons_enable)
-        self.change_button_state(self.chip_button, self.buttons_enable)
+        self.change_button_state(self.kick_button, self.kick_chip_buttons_enable)
+        self.change_button_state(self.chip_button, self.kick_chip_buttons_enable)
 
         # If auto is enabled, we want to populate the autochip or kick message
         if self.auto_kick_button.isChecked():
             self.send_command(ChickerCommandMode.AUTOKICK)
-            self.buttons_should_enable = False
         elif self.auto_chip_button.isChecked():
             self.send_command(ChickerCommandMode.AUTOCHIP)
-            self.buttons_should_enable = False
         elif self.no_auto_button.isChecked():
-            self.buttons_should_enable = True
+            pass
```

Comment:
nit: instead of pass we should just remove this case

Reply:
done